README.md

```diff
@@ -1,15 +1,15 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# To do list
 
-> Description the project.
+> This project is a simple design of a to-do-list using webpack.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML & CSS
+- JavaScript
+- Webpack
 
 ## Live Demo (if available)
 
```

Comment:
- [ ] Let's keep the documentation clean. If you have no live demo link, kindly remove this section.